<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="LB_Fondo">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FONDO</string>
    </property>
   </widget>
   <widget class="QLabel" name="LB_Texto">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TEXTO</string>
    </property>
   </widget>
   <widget class="QLabel" name="LB_BG_Rojo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ROJO</string>
    </property>
   </widget>
   <widget class="QLabel" name="LB_BG_Verde">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>VERDE</string>
    </property>
   </widget>
   <widget class="QLabel" name="LB_BG_Azul">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>AZUL</string>
    </property>
   </widget>
   <widget class="QLabel" name="LB_FG_Rojo">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ROJO</string>
    </property>
   </widget>
   <widget class="QLabel" name="LB_FG_Verde">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>VERDE</string>
    </property>
   </widget>
   <widget class="QLabel" name="LB_FG_Azul">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>AZUL</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="SB_BG_Rojo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="SB_BG_Verde">
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>80</y>
      <width>50</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="SB_BG_Azul">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>80</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="SB_FG_Rojo">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="SB_FG_Verde">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="SB_FG_Azul">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSlider" name="SL_BG_Rojo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="SL_BG_Verde">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>109</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="SL_BG_Azul">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>108</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="SL_FG_Rojo">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>107</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="SL_FG_Verde">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>107</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="SL_FG_Azul">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>107</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="LB_Muestra">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>301</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #000000; color: #FFFFFF; font-family: Arial; font-size: 48px;</string>
    </property>
    <property name="text">
     <string>Muestra</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LB_RES_Fondo">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>430</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FONDO</string>
    </property>
   </widget>
   <widget class="QLabel" name="RES_Fondo">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>430</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>#000000</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_Copiar_Fondo">
    <property name="geometry">
     <rect>
      <x>229</x>
      <y>420</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Copiar Fondo</string>
    </property>
   </widget>
   <widget class="QLabel" name="LB_RES_Texto">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>450</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TEXTO</string>
    </property>
   </widget>
   <widget class="QLabel" name="RES_Texto">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>450</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>#FFFFFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_Copiar_Texto">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>445</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Copiar Texto</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
